packages/picasso.js/src/core/chart-components/ref-line/refline.js

```diff
@@ -20,6 +20,27 @@ function filterUndefinedValue(line) {
   return typeof value !== 'undefined';
 }
 
+function isInvert(scale) {
+  const range = scale.range();
+  return range?.length === 2 && range[0] === 1 && range[1] === 0;
```

Comment:
There is no guarantee that the range is 0 and 1. So I think you have to check if the first value is larger then the last value.

Reply:
ok